ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -37,10 +41,33 @@ pub fn process_bls_to_execution_change<
         BYTES_PER_LOGS_BLOOM,
         MAX_EXTRA_DATA_BYTES,
     >,
-    _signed_address_change: &mut SignedBlsToExecutionChange,
-    _context: &Context,
+    signed_address_change: &mut SignedBlsToExecutionChange,
+    context: &Context,
 ) -> Result<()> {
-    unimplemented!()
+    let address_change = signed_address_change;
```

Comment:
to emulate the spec here, we can take a reference to the inner `message` field; then you can change all the `address_change.message` accesses to just `address_change` below

```suggestion
    let address_change = &signed_address_change.message;
```